flexget/plugins/plugin_configure_series.py

```diff
@@ -17,6 +17,14 @@
 Base = db_schema.versioned_base('import_series', 0)
 
 
+def supported_ids():
```

Comment:
might be good in FilterSeries base so we don't need to copy paste this too much ..